examples/jenkins-master/jenkins-master-template.json

```diff
@@ -0,0 +1,251 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "jenkins-master",
+    "annotations": {
+      "description": "Jenkins service with Kubernetes plugin, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing",
```

Comment:
@bparees two things:

1) installing git plugin
2) copying the sample job

If we can get 1) into official image and 2) we can have a way to install job when you run (like provide GIT repo as env var for `run`,) I can just point to official Jenkins template instead :-)